src/sst/core/configGraph.cc

```diff
@@ -904,45 +922,42 @@ ConfigGraph::getCollapsedPartitionGraph()
     PartitionComponentMap_t& pcomps = graph->getComponentMap();
     PartitionLinkMap_t& plinks = graph->getLinkMap();
 
+
+    // Mark all Components as not visited
+    for ( ConfigComponentMap_t::iterator it = comps.begin(); it != comps.end(); ++it ) it->visited = false;
```

Comment:
Is this needed? It looks like you initialize visited to false in the constructor.

Reply:
Not strictly and I debated on whether to put it in or not.  I thought it safest to put it in since there is technically nothing stopping it from being modified after initialization.